builtin/providers/datadog/resource_datadog_user.go

```diff
@@ -107,7 +108,7 @@ func resourceDatadogUserRead(d *schema.ResourceData, meta interface{}) error {
 		return err
 	}
 
-	log.Printf("[DEBUG] user: %v", u)
+	log.Printf("[DEBUG] user: %s", spew.Sdump(u))
```

Comment:
I just removed the entire debug line as it's pretty much useless :) <3 for the fixing of the debug though